dist/stat_data.py

```diff
@@ -210,9 +214,19 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_pages_inuse',
         'pages currently held in the cache', 'no_clear,no_scale'),
     CacheStat('cache_read', 'pages read into cache'),
+    CacheStat('cache_read_lookaside',
+        'pages read into cache requiring lookaside entries'),
```

Comment:
I don't understand this comment:

> Maybe change this into `lookaside_read` operations and include `lookaside_read_bytes`, grouped along with the inserts.

My intent was to track the pages that require LAS record instantiations. We could subdivide that into "required a search of LAS records" and "required LAS records because the readers still existed"?

Anyway, I'm not sure why we're talking about bytes/inserts here?